update_doc.py

```diff
@@ -122,15 +126,155 @@ def check_supported_models(check: bool):
             f.write(final_doc)
 
 
-def main():
-    parser = argparse.ArgumentParser()
-    parser.add_argument("--check", action="store_true")
+def start_server_and_wait():
+    log_file = open("/tmp/server_log.txt", "w")
 
-    args = parser.parse_args()
+    process = subprocess.Popen(
+        ["text-generation-launcher"],
```

Comment:
Can we use an extremely small model ?

Reply:
Maybe to simplify we could add a flag in `router` in order to print the openapi directly ?
Would remove the necessity to compile things (except the router).

I know I said internally I didn't like that option, but I also said after that you were right since I didn't have a better idea.

Reply:
updated and added a `text-generation-router print-schema` command that prints the api docs as JSON and exits, if no command is passed it goes through the normal flow